src/app/vm/vm-actions/vm-reset-password-component/vm-reset-password.component.html

```diff
@@ -0,0 +1,13 @@
+<div>
+  <div [innerHTML]="translatedMessage | async"></div>
+  <div >
+    <b>{{ 'VM_POST_ACTION.VM_PASSWORD' | translate }}:</b> {{vm.password}}
+    <button md-button color="primary" *ngIf="showSaveButton"
+            (click)="savePassword()">{{ 'COMMON.SAVE' | translate }}</button>
```

Comment:
need to show "Saved" text and prevent further clicks
And, may be we should show check_circle icon instead